extensions/common/aws/aws-s3-core/src/main/java/org/eclipse/edc/aws/s3/S3BucketSchema.java

```diff
@@ -19,11 +19,10 @@ public interface S3BucketSchema {
     String TYPE = "AmazonS3";
     String REGION = "region";
     String BUCKET_NAME = "bucketName";
-    String KEY_NAME = "keyName";
-    String KEY_PREFIX = "keyPrefix";
     String FOLDER_NAME = "folderName";
     String ACCESS_KEY_ID = "accessKeyId";
     String SECRET_ACCESS_KEY = "secretAccessKey";
     String ENDPOINT_OVERRIDE = "endpointOverride";
-    String SECRET_TOKEN = "secretToken";
```

Comment:
ditto

Reply:
This feature in particular didn't have an official release like the others, should I deprecate that either way?

Reply:
ok, yes, this one it can be removed then